llms/mlx_lm/utils.py

```diff
@@ -22,8 +22,10 @@
 }
 
 linear_class_predicate = (
-    lambda m: isinstance(m, nn.Linear) and m.weight.shape[0] % 32 == 0
-)  # TODO remove this once we support quantization for non-multiples of 32
+    lambda m: isinstance(m, nn.Linear)
+    and m.weight.shape[0]
+    != 8  # avoid quantizing gate layers, otherwise we have to re-quant and uplaod all the mixtral models
```

Comment:
👍 at some point we can remove this... but for now I think it's ok.

Nit: typo `uplaod` -> `upload`

Reply:
Thanks for pointing out the typo. I have fixed it :)